OpenAPITools/openapi-generator
# Add source folder variable to fastapi fix 12118

ref: #12118 

### What was done:
- Add Codegen SourceFolder Variable to FastAPI Codegen

### How to test changes:
- Edit soureFolder Variable in bin/configs/python-fastapi.yaml
- generate examples
- check different folder in example


### PR checklist
 
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
@taxpon @frol @mbohlool @cbornet @kenjones-cisco @tomplus @spacether
